examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb

```diff
@@ -106,6 +106,16 @@
     "### Import required libraries and functions"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model.set_retrieval_candidates_for_evaluation(train_tt)
</pre><p>Could you pls remove <code>model.set_retrieval_candidates_for_evaluation(train_tt)</code> line? this command is not valid anymore. We dont need this line now. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>